tests/test_admin.py

```diff
@@ -0,0 +1,98 @@
+from pathlib import Path
+
+from django.contrib.admin import AdminSite
+from django.contrib.auth.models import User
+from django.core.files import File
+from django.test import RequestFactory, TestCase
+from django.utils.translation import gettext_lazy as _
+
+from privates.test import temp_private_root
+from pyquery import PyQuery as pq
+
+from simple_certmanager.admin import CertificateAdmin
+from simple_certmanager.constants import CertificateTypes
+from simple_certmanager.models import Certificate
+
+TEST_FILES = Path(__file__).parent / "data"
+
+
+@temp_private_root()
+class AdminTests(TestCase):
+    def test_list_view(self):
+        """Assert that certificates are correctly displayed in the list view"""
+
+        with open(TEST_FILES / "test.certificate", "r") as client_certificate_f:
+            certificate = Certificate.objects.create(
+                label="Test certificate",
+                type=CertificateTypes.key_pair,
+                public_certificate=File(client_certificate_f, name="test.certificate"),
+            )
+
+        certificate_admin = CertificateAdmin(model=Certificate, admin_site=AdminSite())
+
+        # fake a superuser admin request to changelist_view
+        request = RequestFactory().get("/dummy")
+        request.user = User.objects.create_user(is_superuser=True, username="admin")
+        response = certificate_admin.changelist_view(request)
```

Comment:
this is not the typical pattern to test views - this should ideally be reversing the admin URLs and using `self.client` to make requests.